tests/FindLib3MF.cmake

```diff
@@ -36,7 +36,7 @@ if ("${LIB3MF_LIBDIR}" STREQUAL "")
 endif()
 
 if (NOT ${LIB3MF_LIBDIR} STREQUAL "")
-  set(LIB3MF_LDFLAGS "-L${LIB3MF_LIBDIR}" "-l3MF")
+  set(LIB3MF_LDFLAGS "-L${LIB3MF_LIBDIR}" "-l3MF -lzip -lz")
```

Comment:
Is this an unrelated fix, or accidental?

Reply:
Yep, build fix as I wanted to try exporting 3MF to OctoPrint and the new Debian package strips the packaged libs from the lib3mf dev package.